game.js

```diff
@@ -1,11 +1,65 @@
-let trumps = document.querySelectorAll('.trump');
+/**
+ * Provides random number between min and max provided
+ *
+ * @param min
+ *          Minimum number required
+ * @param max
+ *          Maximum number required
+ * @returns {*}
+ *          Random number
+ */
+function randTime(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+// Available classes for trump position
+var positions = ['animateTop', 'animateLeft', 'animateRight'];
+
+// Available classes for trumps
+var trumps = ['.trump01','.trump02','.trump03','.trump04','.trump05','.trump06']
+
+/**
+ * Provides single random element from supplied array
+ *
+ * @param array
+ *             Source array
+ *
+ * @returns {*}
+ *             Random element
+ */
+function getRandom(array) {
+    var length = array.length;
+    var idx = Math.floor(Math.random() * length);
+    return array[idx];
+}
+
+/**
+ * Adds and removes randomly selected classes to randomly selected trumps
+ *
+ */
+function randTrump() {
+    setInterval(() => {
+        var randomTrump = document.querySelector(getRandom(trumps))
```

Comment:
As the variables trumps and positions are only used inside this one function, it is better practice to keep their definitions inside the function as well. Keeps things easier to read (they are defined closer to where they are used) and keeps variable scope a bit tidier in the code base as a whole!